### PR TITLE
add netdata, the open source real-time performance monitoring for Linux

### PR DESCRIPTION
**Pull request by ktsaou**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +97,7 @@ _Web Analytics_
   * [Countly](https://count.ly) - Real time mobile & web analytics, crash reporting and push notifications platform. ([Source Code](https://github.com/countly)) `AGPLv3` `Javascript`
   * [Druid](http://druid.io/) - A distributed, column-oriented, real-time analytics data store. ([Source Code](https://github.com/druid-io/druid/)) `Apache` `Java`
   * [GoAccess](http://goaccess.io/) - Real-time web log analyzer and interactive viewer that runs in a terminal. ([Source Code](https://github.com/allinurl/goaccess)) `GPLv2` `C`
+  * [netdata](http://my-netdata.io/) - Real-time performance monitoring over the web, with plugins and alarms. ([Source Code](https://github.com/firehol/netdata)) `GPLv3+` `C`
```
Comment (Kickball) [inline README.md]: Looks good, only thing is the license might have to be changed to just GPLv3. (@nodiscc knows licenses best)

Comment (ktsaou) [inline README.md]: But it is `GPLv3+`...

**Comment by nodiscc** [comment]
Hello everyone, sorry I'm not going to merge it since it's already listed in https://github.com/n1trux/awesome-sysadmin#metric--metric-collection, and is a sysadmin tool aimed at machine monitoring. See https://github.com/Kickball/awesome-selfhosted#monitoring.

**Comment by Kickball** [comment]
I missed this the first time, we have already had Netdata suggested in #627. This should be submitted to awesome-sysadmin.

**Comment by nodiscc** [comment]
See https://github.com/Kickball/awesome-selfhosted/issues/127

**Pull request closed without merging**